flow360/component/simulation/operating_condition/operating_condition.py

```diff
@@ -131,10 +131,10 @@ def dynamic_viscosity(self) -> ViscosityType.Positive:
 
     # TODO: should we make this private_attribute?
     @pd.validate_call
-    def _mu_ref(self, mesh_unit: LengthType.Positive) -> pd.PositiveFloat:
+    def _mu_ref(self) -> pd.PositiveFloat:
         """Computes nondimensional dynamic viscosity."""
         # TODO: use unit system for nondimensionalization
-        return (self.dynamic_viscosity / (self.speed_of_sound * self.density * mesh_unit)).v.item()
+        return (self.dynamic_viscosity / (self.speed_of_sound * self.density)).v.item()
```

Comment:
I don't think this change is correct. First, length unit is needed in order to get correct units thus this expression is incorrect. Second, I would expect we remove this function entirely and port conversion to nondimensionalization module